inception-project/inception
# #2379 Push recommender errors via websockets to user

**What's in the PR**
- moved adding to feedbackpanel to reusable classes
- added events for training and selection task errors
- added footer item to subscribe to recommender errors
- added websocket controller for recommender events

**How to test manually**
* make websockets available with `websocket.enabled=true` in the settings.properties
* configure an external recommender with an incorrect url
* open the annotationpage
* an error from the external recommender should be shown
